### ISSUE TITLE
A11y_SQLdatabase projects extension_Advance Publish_Publish Option_Voiceover: Voiceover is not announcing the information appeared under "Option Description" section for respective selected record from "Publish Option" table.

### ISSUE THREAD

**Issue by manaswita-chougule** [microsoft/azuredatastudio#22165]
### GitHub Tags:
#A11yWCAG2.1; #A11ySev2; #A11yTCS; #DesktopApp; #SH-SQL Azure Data Studio-MAC-March23; #SQL Azure Data Studio; #MAC; #AINotInScope; #MAS1.3.1; #Info and Relationships; #Element:Text; #Voiceover;

### Actual Result:​
Voiceover is not announcing the information appeared under "Option Description" section for respective selected record from Publish Option table.

### Expected Result:​
Voiceover should announce the information appeared under "Option Description" section for respective selected record from Publish Option table.

### Environment Details:​
Application Name: SQL Azure Data Studio
Version: 1.42.0-insider (Universal)
MAC Version: macOS Ventura 13.3

### Repro Steps:

1.Start Voiceover with using shortcut key Command+F5.
2.Launch Azure Data Studio
3.Tab till extension and search for “SQL Database Projects” and install it.
4.Tab till database project and press enter.
5.“Database Project” panel will open.
6.Tab till existing project name and press context menu button(right click).
7.Tab till "Publish" menu-item in list and press enter.
8.Tab till "Advance" button and press enter.
9.Tab till any cell and press enter to tick the tick-box.
10."Option Description" will appear.
11.Verify whether Voiceover is not announcing the information appeared under "Option Description" section for respective selected record from "Publish Option" table or not.

### User Impact:
Voiceover user will not be able to get the extra information about the selected record in table and miss the important content of the page.

### Attachment:

![publish opton - description](https://user-images.githubusercontent.com/91586101/223124038-4c8000d2-d80c-4a13-aa1b-9da9930b1090.gif)

**Comment by manaswita-chougule** [comment]
#ReOpened; Regressed:03-17-23;
Verified the issue in below environment:
Application Name: SQL Azure Data Studio
Version: 1.43.0-insider (Universal)
MAC Version: macOS Ventura 13.3

The issue is still repro'ing, hence reopening the bug. Please find attachment for reference. Thanks!
![22165_Repro](https://user-images.githubusercontent.com/91586101/225881839-2d290665-30e5-4d1a-9e75-503963b32087.gif)

**Comment by kisantia** [comment]
reopening since the PR accidentally closed the issue

**Comment by kisantia** [comment]
@manaswita-chougule this has been fixed in the latest version of the sql database projects extension (0.23.0). Please download the [latest insiders build](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch) and verify this is fixed. Thanks!

**Comment by manaswita-chougule** [comment]
#Closed; #Regressed:03-31-23;
Verified the issue in below environment:
Application Name: SQL Azure Data Studio
Version: 1.43.0-insider (Universal)
MAC Version: macOS Ventura 13.3

The issue is fixed, hence closing the bug. Please find below attachment for reference. Thanks!

![22165_Fixed](https://user-images.githubusercontent.com/91586101/229092558-54bcdd54-4c36-4274-b68b-ced386b3b645.gif)
